src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -160,6 +160,17 @@ public class AzureSynapsePdao {
           + "<endif>\n"
           + ") AS rows;";
 
+  private static final String countNullsInTableTemplate =
+      "SELECT COUNT(DISTINCT(datarepo_row_id)) AS rows_with_nulls FROM [<tableName>] WHERE <nullChecks>;";
+
+  private static final String createFinalParquetFilesTemplate =
+      "CREATE EXTERNAL TABLE [<finalTableName>]\n"
+          + "WITH (\n"
+          + "    LOCATION = '<destinationParquetFile>',\n"
+          + "    DATA_SOURCE = [<destinationDataSourceName>],\n"
+          + "    FILE_FORMAT = [<fileFormat>]\n"
+          + ") AS SELECT * FROM [<scratchTableName>] <where> <nullChecks>;";
```

Comment:
We may want to explicitly list columns here to make sure they get serialized in the right order